ir_filters_multi_company/views/ir_filters.xml

```diff
@@ -1,28 +1,30 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8" ?>
 <!-- Copyright 2019 ACSONE SA/NV
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
-
 <odoo>
-
     <record model="ir.ui.view" id="view_ir_filters_form">
         <field name="model">ir.filters</field>
-        <field name="inherit_id" ref="base.ir_filters_view_form"/>
+        <field name="inherit_id" ref="base.ir_filters_view_form" />
         <field name="arch" type="xml">
             <field name="user_id" position="after">
-                <field name="company_id" groups="base.group_multi_company"/>
+                <field
+                    name="company_id"
+                    options="{'no_open': True, 'no_create': True}"
+                    groups="base.group_multi_company"
+                />
             </field>
+            <field name="company_id" invisible="1" />
```

Comment:
Why is this field added a second time, invisible?

Reply:
Which domain?

Reply:
A future one